cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -891,6 +897,8 @@ public MapAndStatus(T loadMap, K status) {
                                         });
                                 } else {
                                     LOGGER.infoCr(reconciliation, "Getting Cruise Control rebalance user task status");
+                                    Set<Condition> conditions = validate(reconciliation, kafkaRebalance);
+                                    conditions.add(StatusUtils.buildWarningCondition("InvalidAnnotation", "Wrong annotation value: " + rebalanceAnnotation(reconciliation, currentKafkaRebalance) + ". The rebalance resource supports the following annotations in 'Rebalancing' state: stop."));
```

Comment:
How do you know there is some rebalance annotation set? The If-else logic before shows that the annotation is not `KafkaRebalanceAnnotation.stop` ... but how do you know it is not null?